### ISSUE TITLE
Banner not showing up when song changes

### ISSUE THREAD

**Issue by alessandrocapra** [citruspi/Spotify-Notifications#85]
I just installed the app, it also showed be the preferences screen but the banner with the song's artist and title is not appearing at all.

I'm a brand new Mac user, is there some special command I need to run to make it appear? I also tried to reboot the pc, but still no luck.

Thanks!

**Comment by mstnorris** [comment]
I am no longer getting updates either. Just a guess but maybe Spotify updated something?

**Comment by sebj** [comment]
That's very odd, both the released version of the app, and the development `swift` branch copy are working fine here. Are there no circumstances under which it can be made to work? (Have we managed to glitch out a checkbox or something in the latest release?) I'm not sure I can think of why it wouldn't be working.

**Comment by mstnorris** [comment]
@sebj apologies, it is now working. I restarted the machine, started spotify, and the notifications, and then quit both, tried again in a different order and nothing I could do was making it work. I just skipped a song and it is now back (previously I was sporadically getting the 'no song info' popup).

I'll keep an eye on it and report back if I can reproduce consistently. Thanks for the swift reply.
